bundles/org.openhab.binding.senechome/src/main/resources/OH-INF/i18n/senechome.properties

```diff
@@ -18,6 +18,8 @@ thing-type.config.senechome.senechome.limitationTresholdValue.label = Limitation
 thing-type.config.senechome.senechome.limitationTresholdValue.description = Treshold in percent, defines when limitation state is enabled
 thing-type.config.senechome.senechome.refreshInterval.label = Refresh Interval
 thing-type.config.senechome.senechome.refreshInterval.description = Rate of refreshing details (in s)
+thing-type.config.senechome.senechome.useHttp.label = Legacy http (Plain)
+thing-type.config.senechome.senechome.useHttp.description = Use http, instead of https
```

Comment:
Please make a choice regarding label/description for your parameter and align i18n and thing description (XML).

Reply:
what do you mean with this?
Files are aligned with spotless.

Reply:
Label is "Http access" in XML while "Plain http (Legacy)" in the i18n file.
Description is "Use legacy http access" in XML while "Use http, instead of https" in he i18n file.

Reply:
For the label, the good practice is a short label with only 2 or 3 words with each word starting with a character in uppercase. No parenthesis. Maybe something like "Use HTTP"

Reply:
ok got it